farsi_transcribe/utils.py

```
@@ -0,0 +1,292 @@
+"""
+Utility classes and functions for FarsiTranscribe.
+
+This module provides helper classes for result management, text processing,
+and output formatting.
+"""
+
+import json
+import logging
+from pathlib import Path
+from typing import List, Dict, Any, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)